code-challenges.js

```diff
@@ -39,3 +84,10 @@ var numbersToAdd2 = [0, 7, -8, 12]
 // Expected output: [0, 7, -1, 11]
 var numbersToAdd3 = []
 // Expected output: []
+
+let addedNumbers = (array)  => { 
+    let newArray = []
+    array.reduce(function(a,b,i) { return newArray[i] = a+b; },0)
+    return newArray
+}
+    console.log(addedNumbers(numbersToAdd1))
```

Comment:
Awesome! I think its important that you also show you checked for the other two arrays by having 3 different console.logs. 

In an interview, take home challenge, this helps communicate that you tested your functionality on all variables. 

Otherwise SUPER good job!